modules/geo-layers/src/mvt-layer/coordinate-transform.js

```diff
@@ -0,0 +1,56 @@
+import {lerp} from 'math.gl';
+
+const availableTransformations = {
```

Comment:
I do not believe the MVTLoader ever generates Multi* features:

https://github.com/visgl/loaders.gl/blob/9a62f53206db982858a42bdfde4eb1ea9606aadf/modules/mvt/src/lib/transform-to-local-range.js#L1

Reply:
We are rendering Multi* features.. :roll_eyes:
https://github.com/visgl/loaders.gl/blob/9a62f53206db982858a42bdfde4eb1ea9606aadf/modules/mvt/src/lib/transform-to-local-range.js#L61